iree/compiler/Dialect/Vulkan/Utils/TargetTriple.cpp

```diff
@@ -207,6 +222,30 @@ CapabilitiesAttr getCapabilities(const TargetTriple &triple,
       storageBuffer8BitAccess = true, storagePushConstant8 = true;
       uniformAndStorageBuffer8BitAccess = true;
 
+      variablePointers = variablePointersStorageBuffer = true;
+      break;
+    case TargetTripleArch::Apple_M1:
+      // Example: https://vulkan.gpuinfo.org/displayreport.php?id=10312
```

Comment:
nit: this refers to a MALI devices. We probably want to update to 
https://vulkan.gpuinfo.org/displayreport.php?id=14522

Reply:
Oops. Good catch! I thought I changed the URL. But somehow messed up. Fixed.